app/api/v1/dpm_endpoints.py

```diff
@@ -74,3 +75,30 @@ async def download_file(
         )
     except Exception as exc:
         raise HTTPException(status_code=500, detail=str(exc)) from exc
+
+@router.get("/list-files", response_model=List[str])
```

Comment:
```suggestion
@router.get("/list", response_model=List[str])
```
How do you feel about just "list" for simplicity